tests/conftest.py

```diff
@@ -7,6 +7,13 @@ def dataset():
     mnist = tf.keras.datasets.mnist
     (x_train, y_train), (x_test, y_test) = mnist.load_data()
     x_train, x_test = x_train / 255.0, x_test / 255.0
+
+    x_train = x_train[:3000, :, :]
```

Comment:
Minor nit:

Following should work and is nicer to read.
```python
x_train = x_train[:3000]
y_train = y_train[:3000]
```

Reply:
For now it's just experimenting to see why e2e tests are not passing and what can be done about it. But thanks anyway :smile: